drizzlepac/mapreg.py

```diff
@@ -125,6 +356,85 @@ def map_region_files(input_reg, images, img_wcs_ext='sci',
+        the ``@`` syntax should be expanded before calling this helper. Entries
+        may be ``None`` to skip a slot.
+    images : str or sequence of str
+        FITS image filenames (or wildcards that have already been expanded)
+        that provide the WCS information used for the transformation.
+    img_wcs_ext : str or sequence of str or tuple, optional
+        FITS extension specifiers identifying which extensions in ``images``
+        should receive transformed regions. Strings are interpreted as
+        extension names and expanded to all available EXTVER values. Tuples of
+        ``(extname, extver)`` or integers (extension numbers) are respected as
+        given. Default is ``'sci'``.
+    refimg : str or None, optional
+        Reference FITS image that supplies WCS information when any input
+        region is provided in image-like coordinates. May include an extension
+        specifier. Required when ``input_reg`` contains image-coordinate
+        regions. Default is ``None``.
+    ref_wcs_ext : str or sequence of str or tuple, optional
+        Extension specifier(s) for selecting the WCS headers from ``refimg``
+        that correspond to each entry in ``input_reg``. The interpretation
+        mirrors ``img_wcs_ext``. Default is ``'sci'``.
+    chip_reg : str or sequence of str or None, optional
+        Region file(s) already in image coordinates that should be appended to
+        every transformed region file. The list is matched one-to-one with
+        ``img_wcs_ext`` entries; ``None`` skips a position. Default is ``None``.
+    outpath : str, optional
+        Directory where transformed region files (and optional catalog) will be
+        written. Created files are named ``<image><ext>_twreg.reg``. Default is
+        ``'./regions'``.
+    filter : {None, 'fast', 'precise'}, optional
+        Method for removing regions that fall fully outside the target image.
+        ``'fast'`` performs bounding-box clipping; ``'precise'`` currently
+        downgrades to ``'fast'``. ``None`` disables filtering. Default is
+        ``None``.
+    catfname : str or None, optional
+        Name of an exclusions catalog to write summarizing the generated region
+        files. When ``None`` no catalog is produced. Default is ``None``.
+    iteractive : bool, optional
+        Reserved for future interactive workflows. Currently unused. Default
+        is ``False``.
+    append : bool, optional
+        If ``True`` and an output region file already exists, append the newly
+        transformed regions instead of overwriting the file. Default is
+        ``False``.
+    verbose : bool, optional
+        When ``True`` print additional progress information and associations
+        between extensions and chip regions. Default is ``True``.
+
+    Returns
+    -------
+    None
+        All results are persisted to ``outpath`` and optionally to the catalog
+        file.
+
```

Comment:
No need to say "return None" but you will have to move the note about all results elsewhere, maybe the top.

```suggestion
```